hibernate-core/src/main/java/org/hibernate/loader/ast/internal/LoaderSelectBuilder.java

```diff
@@ -207,6 +209,31 @@ public static SelectStatement createSubSelectFetchSelect(
 	private int fetchDepth;
 	private Map<OrderByFragment, TableGroup> orderByFragments;
 
+	private LoaderSelectBuilder(
+			SqlAstCreationContext creationContext,
+			Loadable loadable,
+			List<? extends ModelPart> partsToSelect,
+			List<ModelPart> restrictedParts,
```

Comment:
Why there is inconsistency above for the two list (one uses bounded generic type whereas the other doesn't)? That would confuse the future code reader always.